crates/sovereign-sdk/full-node/db/sov-db/src/ledger_db/mod.rs

```diff
@@ -130,6 +131,60 @@ impl LedgerDB {
         })
     }
 
+    /// Drop a column family from the database
+    pub fn drop_cf(
+        cfg: &RocksdbConfig,
+        column_families: Option<Vec<String>>,
+        cf_name: &str,
+    ) -> anyhow::Result<()> {
+        let path = cfg.path.join(LEDGER_DB_PATH_SUFFIX);
+        let raw_options = cfg.as_raw_options(false);
+        let mut inner = DB::open(
+            path,
+            "ledger-db",
+            column_families
+                .unwrap_or_else(|| LEDGER_TABLES.iter().map(|s| s.to_string()).collect()),
+            &raw_options,
+        )?;
+
+        inner.drop_cf(cf_name)?;
+
+        Ok(())
+    }
+
+    /// List all column families in the database
+    pub fn list_column_families(path: &Path) -> Vec<String> {
+        rocksdb::DB::list_cf(
+            &rocksdb::Options::default(),
+            path.join(LEDGER_DB_PATH_SUFFIX),
+        )
+        .unwrap()
+    }
```

Comment:
These don't belong to `LedgerDB`... maybe add them as migration helpers / utils?

Reply:
How about I put them in LedgerDBMigrator instead of utils or sth like that?